File: frontend/src/app/shared/models/issues/issue-message.details.ts
import { StackTrace } from '@shared/models/issues/stack.trace';
import { HttpResponseErrorMessage } from '@shared/models/issues/http-response.message';
import { IssueEnvironment } from '@shared/models/issues/issue-environment';

export interface IssueMessageDetails {
    url: string,
    errorMessage: string,
    className: string,
    stackTrace?: StackTrace[],
    responseErrorMessage?: HttpResponseErrorMessage,
    environmentMessage: IssueEnvironment,
}

File: backend/Watchdog.Core/Watchdog.Core.Common/Models/Issue/IssueMessageDetails.cs
using System.Collections.Generic;

namespace Watchdog.Core.Common.Models.Issue
{
    public class IssueMessageDetails
    {
        public string Url { get; set; }
        public string ErrorMessage { get; set; }
        public string ClassName { get; set; }
        public ICollection<StackFrame> StackTrace { get; set; }
        public HttpResponseErrorMessage ResponseErrorMessage { get; set; }
        public IssueEnvironment EnvironmentMessage { get; set; }
        public ICollection<Breadcrumb> Breadcrumbs { get; set; }
    }
}